antlr-kotlin-runtime/src/commonMain/kotlin/org/antlr/v4/kotlinruntime/tree/xpath/XPathElement.kt

```diff
@@ -9,12 +9,13 @@ public abstract class XPathElement(protected var nodeName: String) {
   public var invert: Boolean = false
 
   /**
-   * Given tree rooted at [t] return all nodes matched by this path element.
+   * Given tree rooted at [t], returns all nodes matched by this path element.
    */
   public abstract fun evaluate(t: ParseTree): Collection<ParseTree>
 
-  public override fun toString(): String {
+  override fun toString(): String {
```

Comment:
Why removing public here?

Reply:
@ftomassetti the public visibility is redundant when overriding a public method. There are other 6/7 places where I accidentally created that redundancy, but it was out of scope.

Reply:
I think public is always redundant (as it is the default), but that in case of a library you wanted to make it explicit. I am not sure I understand why it is different for overriden methods

Reply:
@ftomassetti `public` is not redundant in explicit API mode. But it is redundant and not required for definitions that have been declared `public` on the superclass or interface. Example: `Any.toString` is public already, so the visibility modifier is redundant here. If it wasn't redundant, the explicit API checker would emit a warning.